### PR TITLE
Enable rasp blocking tests

### PR DESCRIPTION
**Pull request by uurien**
## Motivation

Blocking for rasp ssrf rules has been merged in dd-trace-js, we should enable the system tests

## Changes

Enable the system tests for next dd-trace-js version. 
As in nodejs we get url with trailing `/`, we are modifying the test to add it in nodejs tests.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [x] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [x] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Comment by uurien** [comment]
re-running failed tests, they seem unrelated

**Review by iunanua — APPROVED** [review]

**Review by cbeauchesne — APPROVED** [review]
Java failures are not related (#2810)

File: tests/appsec/rasp/test_ssrf.py [hunk 1]
```diff
@@ -39,11 +43,15 @@ def setup_ssrf_post_urlencoded(self):
     def test_ssrf_post_urlencoded(self):
         assert self.r.status_code == 403
 
+        expected_http_value = "http://169.254.169.254"
+        if context.library == "nodejs":
+            expected_http_value += "/"
```
[the remark this answers is not in the document]
Comment (simon-id) [reply tests/appsec/rasp/test_ssrf.py:48]: should we trim the trailing slash in our library ? is this spec compliant ?

**Review by simon-id — COMMENTED** [review]
I'd like to not have a special case for the trailing slash, either fix it in our library or have a wildcard in the tests, otherwise LGTM

**Pull request merged**